radio/src/targets/horus/CMakeLists.txt

```diff
@@ -85,7 +85,19 @@ if (PCB STREQUAL X10)
     add_definitions(-DRADIO_FAMILY_T16)
     option(INTERNAL_MODULE_MULTI "Support for MULTI internal module" ON)
     option(BLUETOOTH "Support for bluetooth module" OFF)
+    set(HARDWARE_TOUCH YES)
+    set(USB_CHARGER YES)
     add_definitions(-DMANUFACTURER_JUMPER)
+  elseif (PCBREV STREQUAL TX18S)
```

Comment:
Please keep that for a separate PR, knowing we don't add support for things when we don't have the hardware to test.